include/astshim/base.h

```diff
@@ -78,6 +78,11 @@ Reshape a vector as a 2-dimensional array that shares the same memory
 @warning You must hold onto the original vector until you are done
 with the returned array, else the array will be corrupted.
```

Comment:
Per the [style guide](https://developer.lsst.io/docs/cpp_docs.html#common-structure-of-documentation-blocks), notes should appear after the parameter/return/exception section.